pkg/web/templates/partials/account/account_list.html

```diff
@@ -22,8 +22,8 @@
           {{ else }}
           <td class="px-16">-</td>
           {{ end }}
-          {{ $created := .Created.Format "January 2, 2006 15:04:05 -0700" }}
-          <td>{{ $created }}</td>
+          {{ $created := .Created.Format "2006-01-02T15:04:05-0700" }}
+          <td class="datetime">{{ $created }}</td>
```

Comment:
Basically `updateDatetimes` looks for elements with this class, then converts it to a localized timestamp, removing the class when it's done so that it is not executed multiple times on accident.